tests/test_effects.py

```diff
@@ -180,3 +180,10 @@ def test_weak(self):
 
         attack_phase = get_attack(fish, t3[0].pet)
         self.assertEqual(attack_phase[1], 9) # 6/8 + 3
+
+    def test_peacock(self):
```

Comment:
It would be good if the test made sure that it triggers the correct number of times